pkg/neorpc/filters.go

```diff
@@ -126,14 +147,76 @@ func (f *NotificationFilter) Copy() *NotificationFilter {
 		res.Name = new(string)
 		*res.Name = *f.Name
 	}
+	if len(f.Parameters) != 0 {
+		res.Parameters = slices.Clone(f.Parameters)
+	}
+	f.parametersCache = nil
 	return res
 }
 
+// ParametersSI returns [stackitem.Item] version of [NotificationFilter.Parameters]
+// according to [smartcontract.Parameter.ToStackItem]; Notice that the result is
+// cached internally in [NotificationFilter] for efficiency, so once you call this
+// method it will not change even if you change any structure fields. If you need
+// to update parameters, use [NotificationFilter.Copy].
+// It mainly should be used by server code. Must not be used concurrently.
+func (f *NotificationFilter) ParametersSI() ([]stackitem.Item, error) {
```

Comment:
BTW, this name was just an example, you may use any other name if you'd like to.

Reply:
Good choice.